lib/compat/wordpress-5.9/edit-site-page.php

```diff
@@ -122,6 +122,18 @@ static function( $classes ) {
 		'__unstableHomeTemplate'   => gutenberg_resolve_home_template(),
 	);
 
+	/**
+	 * Filter to enable adding non-REST block patterns to editor settings.
+	 *
+	 * @since 6.0.0
+	 *
+	 * @param bool $should_add_to_settings
+	 */
+	if ( apply_filters( 'should_add_block_patterns_to_editor_settings', false ) ) {
```

Comment:
```suggestion
	if ( apply_filters( 'gutenberg_add_block_patterns_to_editor_settings', false ) ) {
```

:1234: Would need to be renamed throughout